### ISSUE TITLE
Almost impossible to use documentation!

### ISSUE THREAD

**Issue by enjoysmath** [gtkd-developers/GtkD#51]
I need to create a VBox, but the documentation here:
http://api.gtkd.org/src/gtk/VBox.html
says it's deprecated and to use Box instead, passing in 
GTK_ORIENTATION_VERTICAL.

However, upon search all the source files for occurrences of

GTK_ORIENTATION_VERTICAL

I found no info on what it's D equivalent is.  Do you have to be the creator of GtkD to actually use it or what?  This is ridiculous.

**Comment by mtpain** [comment]
Agreed. I don't know GTK, but I love D and need to make a thing with a user interface, so I need more help than API docs, like a quickstart and tutorial. I know it's not trivial, but just thought I'd let you know one more person is interested. Thanks for your hard work!

**Comment by MikeWey** [comment]
Steve Tale wrote a few articles on how to get started with GtkD: http://britseyeview.com/software/articles/gsgtkd.html

**Comment by mtpain** [comment]
Excellent, thanks!

**Comment by 9898287** [comment]
> Okay, did a search for "ORIENTATION" and found it:
> 
> public enum GtkOrientation
> {
> HORIZONTAL,
> VERTICAL
> }
> 
> Lol

I have the same problem. What module does this enum even belong to? Can't find it in the official docs.

**Comment by 9898287** [comment]
Nevermind I found it.
I ended up writing a small tool to search keywords recursively in a given directory and ran in ~/dub/gtk-d

$ termsearch -t "public enum GtkOrientation" -v
1: L:1796: F:"generated/gtkd/gtk/c/types.d":
public enum GtkOrientation

Total match found:	1
Total files checked:	1292
Total files ignored:	29

So I looked for the corresponding entry in the documentation and found this:
https://api.gtkd.org/gtkd/gtkc/gtktypes.html

lol
